Orange/widgets/visualize/owboxplot.py

```diff
@@ -18,6 +17,7 @@
 from scipy.stats import f_oneway, chi2_contingency
 
 from Orange.data import Table
+from Orange.data.dask import maybe_compute
```

Comment:
Mi lahko razlozil, zakaj se tale `maybe_compute` sploh potrebuje, ce itak stolpce pretvarjamo v numpy array? Kje je tista caka, ki je ne vidim?

Reply:
Se strinjam, idealno bi bilo, da se tega znebimo. Zaenkrat sem funkcijo pustil, samo prestavil sem jo tja, kjer imamo DaskTable, da se jo lahko importa, ker je prakticna za prototipe.

V boxplotu jo rabim ker se za diskretne statistike racuna ditribution in contingency, ki vracata dask. Ko uspem to dvoje zrihtat pa lahko se maybe compute odstranim.